### ISSUE TITLE
For DetailList, Narrator is not reading corresponding column headers along with the description of the table cells data.

### ISSUE THREAD

**Issue by chakradhar2** [OfficeDev/office-ui-fabric-react#6019]
Expected behaviour: 
For DetailList, Narrator should read corresponding column headers along with the description of the table cells data.

**Comment by KevinTCoughlin** [comment]
> 
> 
> Expected behaviour:
> For DetailList, Narrator should read corresponding column headers along with the description of the table cells data.

@chakradhar2 can you please provide more context as per our "Bug" template which this _looks_ to be? Primarily which browser(s) and version(s) of OUFR are affected. Bug template below.

```md
<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨

We’re here to help, but we need a few things from you! 

Please help us reduce duplicates by doing the following steps before logging an issue:
  * Search: https://github.com/OfficeDev/office-ui-fabric-react/search?type=Issues
  * Search by area or component: https://github.com/OfficeDev/office-ui-fabric-react/issues/labels
  
Please provide a reproduction of the bug in a codepen, if possible. Here’s how:
  * Goto https://aka.ms/fabricpen for a starting codepen 
  * You can also use the "Export to Codepen" feature for the various components in our documentation site.
  * Alternatively, you can also use https://aka.ms/fabricdemo to get permanent repro links if the repro occurs with an example. 
    (A permanent link is preferable to "use the website" as the website can change)
 
Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
-->

### Environment Information 
 
 <!-- 
  ** If using the website, note that the website is often behind. You can find the current version the website uses at the home page (i.e. at the landing page of uifabric.io).
 -->
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 
<!-- Goto https://aka.ms/fabricpen for a starting codepen -->
<!-- See http://codepen.io/dzearing/pens/public/?grid_type=list for a variety of examples -->
<!-- Alternatively, you can also use https://aka.ms/fabricdemo to get permanent repro links. -->
 
#### Actual behavior:
 
<!-- fill this out -->
 
#### Expected behavior:
 
<!-- fill this out -->

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

```

Doing so will help the team and myself diagnose and provide a fix once confirmed for the affected version(s) of OUFR. Thank you.

**Comment by natalieethell** [comment]
Yep, this is fixed for Narrator! There's an [issue assigned to me for fixing it in JAWS](https://github.com/OfficeDev/office-ui-fabric-react/issues/7123).

Here's what the Narrator output looks like for the [DetailsList Basic Example](https://developer.microsoft.com/en-us/fabric#/components/detailslist/basic):

**<mark name="0 Id:6"/>non-selected, read-only, Column Header Name,, Column <say-as interpret-as="cardinal">2</say-as> of <say-as interpret-as="cardinal">3</say-as> , Item <mark name="1 Id:6"/>0 <mark name="ContinuousReadingEnd Id:6"/>**

**Comment by natalieethell** [comment]
If you are no longer experiencing this issue, @chakradhar2, feel free to close it! It should have been fixed in Narrator (as a result of these two PRs: https://github.com/OfficeDev/office-ui-fabric-react/pull/5925, https://github.com/OfficeDev/office-ui-fabric-react/pull/5190) and we have a [separate issue tracking the JAWS behavior](https://github.com/OfficeDev/office-ui-fabric-react/issues/7123).

Let us know if you have any other questions.

**Comment by chakradhar2** [comment]
> Yep, this is fixed for Narrator! There's an [issue assigned to me for fixing it in JAWS](https://github.com/OfficeDev/office-ui-fabric-react/issues/7123).
> 
> Here's what the Narrator output looks like for the [DetailsList Basic Example](https://developer.microsoft.com/en-us/fabric#/components/detailslist/basic):
> 
> **non-selected, read-only, Column Header Name,, Column 2 of 3 , Item 0 **

@natalieethell: Sorry for late reply.
It is reading same way as you mentioned. But when we select the row,  it still reads same way as earlier, I think it should read with header there also.

| Name           |  Value |
| ------------- | ------------- |
|  Item1   | 1 |
| Item2  | 2 |

If  first row is selected, Currently it reads like:  Item1 , 1 selected
 It should read row some thing like :   **Item1,  Column Header Name, 1 Column header  Value.**  

It should also mention link if present, while reading the row. Suppose Item1 has link
**Item1 Link,  Column Header Name, 1 Column header  Value.**  
 
 If row is read in this way on row selection then it will make more sense.  

@natalieethell :  shall I reopen the Bug or Create a new bug?

**Comment by natalieethell** [comment]
We are currently tracking the bug in issue #7560.

**Comment by natalieethell** [comment]
The discussion moved over to #7560. Going to remove the "Needs: Attention" label
